openghg/standardise/footprints/_acrg_org.py

```diff
@@ -228,7 +235,16 @@ def parse_acrg_org(
     # This might seem longwinded now but will help when we want to read
     # more than one footprints at a time
     # TODO - remove this once assign_attributes has been refactored
-    key = "_".join((site, domain, model, inlet))
+    key_parts = [
```

Comment:
In the short term, for this we'd likely want either:
 - (site, domain, model, inlet)
 - (satellite, obs_region, domain, model, inlet)